bedrock/base/templates/includes/google-analytics.html

```diff
@@ -5,16 +5,6 @@
 <!-- Site Container: NONE -->
 
 {% if settings.GTM_CONTAINER_ID %}
-<script>
-// If doNotTrack is not enabled, it is ok to add GTM
-// @see https://bugzilla.mozilla.org/show_bug.cgi?id=1217896 for more details
-if (!_dntEnabled()) {
-  (function(w,d,s,l,i,j,f,dl,k,q){
-    w[l]=w[l]||[];w[l].push({'gtm.start': new Date().getTime(),event:'gtm.js'});f=d.getElementsByTagName(s)[0];
-    k=i.length;q='//www.googletagmanager.com/gtm.js?id=@&l='+(l||'dataLayer');
-    while(k--){j=d.createElement(s);j.async=!0;j.src=q.replace('@',i[k]);f.parentNode.insertBefore(j,f);}
-  }(window,document,'script','dataLayer',['{{settings.GTM_CONTAINER_ID}}']));
-}
-</script>
+  <script type="text/javascript" src="{{ static('js/base/gtm-snippet.js') }}"></script>
```

Comment:
I think this is a good suggestion (as long as minifying the already minified snippet code doesn't cause any issues)